apps/wallet/src/ui/app/hooks/useGetValidators.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { isSuiObject, isSuiMoveObject } from '@mysten/sui.js';
+import BigNumber from 'bignumber.js';
+import { useMemo } from 'react';
+
+import { useGetObject } from './useGetObject';
+
+//TODO: Remove when available on SDK, types should come from the SDK
+import type {
+    Validator,
+    ValidatorState,
+} from '_app/staking/home/ValidatorDataTypes';
+
+export const VALIDATORS_OBJECT_ID = '0x05';
+
+function processValidators(
+    set: Validator[],
+    totalStake: bigint,
+    current_epoch: number,
+    walletAddress?: string | null
+) {
+    return set.map((av) => {
+        const rawName = av.fields.metadata.fields.name;
+
+        const name = Buffer.from(rawName, 'base64').toString();
```

Comment:
There's new name processing logic that recently landed that you need to follow (it's annoying while we transition some data types), you can see it here: https://github.com/MystenLabs/sui/blob/main/apps/wallet/src/ui/app/staking/usePendingDelegation.tsx#L90-L99

Reply:
done